src/app/organization/offices/edit-office/edit-office.component.html

```diff
@@ -0,0 +1,43 @@
+<div class="container">
```

Comment:
Dilienate the HTML.

Reply:
> Dilienate the HTML.

Aligned the space between tags. Please have a look!